ts/features/wallet/store/reducers/index.ts

```diff
@@ -3,17 +3,22 @@ import { PersistPartial } from "redux-persist";
 import preferencesPersistor, { WalletPreferencesState } from "./preferences";
 import cardsReducer, { WalletCardsState } from "./cards";
 import placeholdersPersistor, { WalletPlaceholdersState } from "./placeholders";
+import bottomSheetPersistor, {
+  WalletBottomSheetSurveyState
+} from "./bottomSheet";
 
 export type WalletState = {
   cards: WalletCardsState;
   preferences: WalletPreferencesState & PersistPartial;
   placeholders: WalletPlaceholdersState & PersistPartial;
+  bottomSheet: WalletBottomSheetSurveyState & PersistPartial;
 };
 
 const walletReducer = combineReducers({
   cards: cardsReducer,
   preferences: preferencesPersistor,
-  placeholders: placeholdersPersistor
+  placeholders: placeholdersPersistor,
+  bottomSheet: bottomSheetPersistor
```

Comment:
This reducer is strictly used for the `wallet` feature, which is the main wallet section of the IO App used by payments, bonus and IT-Wallet. 

We should move the logic within the `itWallet` feature